### PR TITLE
factor out common import controller code

### PR DESCRIPTION
**Pull request by malaffoon**
There is already some duplication between Exam and Package import controllers. And we anticipate more controllers (accessibility codes, districts, schools). So i pulled out the common functionality.

### CODE REVIEW

**Comment by malaffoon** [comment]
Sigh, not sure how RDW_Schema got in here ... i was really friggin' careful about refreshing before branching. I think i really don't like git submodules.

File: import-service/src/main/java/org/opentestsystem/rdw/ingest/web/ExamController.java [hunk 1]
```diff
@@ -19,135 +17,66 @@
...
-        return assembler.toResource(service.importContent(sbacUser, body, ImportContent.EXAM, contentType, requestParams));
+        return super.postImport(sbacUser, contentType, body, requestParams);
     }
 
     @PostMapping(value = "/imports", consumes = "multipart/form-data")
     @ResponseStatus(HttpStatus.ACCEPTED)
     public RdwImportResource uploadImport(@AuthenticationPrincipal final SbacUserDetails sbacUser,
                                           final MultipartHttpServletRequest request) {
-        // get the best MultipartFile (either the only one or the one named "file")
-        final MultipartFile file;
```
Comment (chdietz) [inline import-service/src/main/java/org/opentestsystem/rdw/ingest/web/ExamController.java]: So, you created common logic to handle support for posting body and file upload?

File: import-service/src/main/java/org/opentestsystem/rdw/ingest/web/ReportingPackageController.java [hunk 2]
```diff
@@ -27,14 +31,11 @@
...
-class ReportingPackageController {
-
-    private final ImportService service;
-    private final ReportingPackageImportResourceAssembler assembler = new ReportingPackageImportResourceAssembler();
+class ReportingPackageController extends ImportController {
 
     @Autowired
     public ReportingPackageController(final ImportService service) {
-        this.service = service;
+        super(service, ImportContent.PACKAGE, new ImportResourceAssembler(ReportingPackageController.class));
```
Comment (chdietz) [inline import-service/src/main/java/org/opentestsystem/rdw/ingest/web/ReportingPackageController.java]: now package loader can support file upload?

File: import-service/src/main/java/org/opentestsystem/rdw/ingest/web/ExamController.java (hunk 1 above)
Comment (malaffoon) [inline import-service/src/main/java/org/opentestsystem/rdw/ingest/web/ExamController.java]: That was the main driving force, yes. But also the payload and payload/properties end-points.

File: import-service/src/main/java/org/opentestsystem/rdw/ingest/web/ReportingPackageController.java (hunk 2 above)
Comment (malaffoon) [inline import-service/src/main/java/org/opentestsystem/rdw/ingest/web/ReportingPackageController.java]: Yup.

Comment (rcordeau) [inline import-service/src/main/java/org/opentestsystem/rdw/ingest/web/ReportingPackageController.java]: Yeah they all will - Accommodation codes as well.

**Comment by rcordeau** [comment]
I need this update, so I'll merge it in.

**Pull request merged**